src/smsfusion/_ins.py

```diff
@@ -0,0 +1,273 @@
+import numpy as np
+
+from ._transforms import (
+    _angular_matrix_from_euler,
+    _rot_matrix_from_euler,
```

Comment:
Kan disse erstattes med å bruke `Rotation` klassen i scipy? Eller bør vi implementere disse selv for å gjøre koden mer effektiv?

Reply:
Tror det er litt overhead is SciPy sin løsning. Vi kan bruke våre egne "lean" implementasjoner for interne kall. Så bør vi tenke på om vi skal bruke scipy for eksterne apier, eks. `Rotation3D`.